Packages/com.unity.inputsystem/Documentation~/Testing.md

```diff
@@ -173,4 +173,4 @@ Alternatively, arbitrary input events can be fed into the system and arbitrary i
     }
 ```
 
->NOTE: For reference, the tests for the input system itself can be found [here](https://github.com/Unity-Technologies/InputSystem/tree/stable/Packages/com.unity.inputsystem/Tests/InputSystem).
+>NOTE: For reference, the tests for the input system itself can be found [here](https://github.com/Unity-Technologies/InputSystem/tree/stable/Assets/Tests/InputSystem).
```

Comment:
I think the assembly setup here has also changed. Think both the screenshot and the test still refer to the obsolete old name (`Unity.InputSystem.Tests.dll`?).

Reply:
Will get to this when updating screenshots